src/core/include/openvino/core/model.hpp

```diff
@@ -24,9 +24,15 @@
 #include "openvino/runtime/tensor.hpp"
 
 namespace ov {
+namespace frontend {
+class FrontEndUtils;
+}
```

Comment:
can it be a `FrontEnd` itself?

Reply:
Yes, the other approach is to define private class function. Just didn't want declare this in public frontend.hpp header (even if function itself is private). Will do this